public/css/main.css

```diff
@@ -0,0 +1,934 @@
+/*&&&&&&&&&&&&&&&&&&&&&&&&&&
+
+    General styles
+
+&&&&&&&&&&&&&&&&&&&&&&&&&&&*/
+html {
+  font-family: Helvetica, sans-serif;
+  font-size: 1em;
+}
+body {
+  width: 100%;
+}
+article {
+  clear: left;
+  padding: 2em 2em 1em 2em;
+}
+p {
+  line-height: 1.5;
+  letter-spacing: 0.02em;
+}
+a {
+  text-decoration: none;
+}
+button {
+  border: none;
+  background-color: #fd6632;
+  color: white;
+  padding: 1em 1.5em;
+  font-size: 1em;
+}
+button:hover {
+  cursor: pointer;
+}
+article h1, .services h3, .about h2, .clients h2 {
+  position: relative;
+  font-size: 1.777em;
+  overflow: hidden;
+  text-align: center;
+  letter-spacing: 0.1em;
+}
+.flex-container {
+  /*adapted from flexboxin5.com*/
+  display: -webkit-box;      /* OLD - iOS 6-, Safari 3.1-6 */
+  display: -moz-box;         /* OLD - Firefox 19- (buggy but mostly works) */
+  display: -ms-flexbox;      /* TWEENER - IE 10 */
+  display: -webkit-flex;     /* NEW - Chrome */
+  display: flex;             /* NEW, Spec - Opera 12.1, Firefox 20+ */
+}
+.nav ul {
+  list-style-type: none;
+}
+.nav a {
+  color: rgb(124,131,135);
+}
+.nav a:hover {
+  color: rgb(0,0,0);
+}
+/* General styles desktop layout */
+@media screen and (min-width: 50em) {
+  body {
+    max-width: 80em;
+    width: 90%;
+    margin: 0 auto;
+  }
+  article {
+    padding: 2em 8%;
+  }
+  article h1, .services h3, .about h2, .clients h2 {
+    font-size: 2.369em;
+  }
+}
+/*&&&&&&&&&&&&&&&&&&&&&&&&&&
+
+    Mock strikethrough styles
+
+&&&&&&&&&&&&&&&&&&&&&&&&&&&*/
+article h1:before, article h1:after, .services h3:before, .services h3:after, .about h2:before, .about h2:after, .clients h2:before, .clients h2:after {
+  position: absolute;
+  top: 51%;
+  overflow: hidden;
+  width: 50%;
+  height: 1px;
+  content: '\a0';
```

Comment:
Interesting choice, going with pseudo-content. I would have used a border, but there are many ways to skin this cat.